rosidl_parser/package.xml

```diff
@@ -12,6 +12,7 @@
   <test_depend>ament_cmake_pytest</test_depend>
   <test_depend>ament_lint_auto</test_depend>
   <test_depend>ament_lint_common</test_depend>
+  <test_depend>pytest</test_depend>
```

Comment:
This rosdep key was wrong. Fixed in follow up commit c9e206ef47a65979a377d01140b70270042715cd.